### PR TITLE
Bumps Go 1.17 version to latest patch version

### PR DESCRIPTION
**Pull request by irbekrm**
Bumps Go 1.17 version to latest patch version

Signed-off-by: irbekrm <irbekrm@gmail.com>

### CODE REVIEW

**Comment by irbekrm** [comment]
/hold

I would like to merge this once trusted cluster has been upgraded to use containerd to test that change

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve

Best to be really explicit about which minor version  we use, I think :grin:  Thanks!

**Pull request merged**
